app/models/article.rb

```diff
@@ -35,7 +35,9 @@ def name
   end
 
   def path
-    if published?
+    if published? && short_path
```

Comment:
And no worries about docs, I understand. This codebase is really well written, I feel like i'm going to learn a lot helping out 💯